index.js

```diff
@@ -31,24 +47,39 @@ FocusGroup.prototype._handleKeydownEvent = function(event) {
   var activeElementIndex = this._getActiveElementIndex();
   if (activeElementIndex === -1) return;
 
-  var arrow = getEventArrowKey(event);
+  var keyCode = event.keyCode
+  var keybindings = this._settings.keybindings
 
-  if (!arrow) {
-    this._handleNonArrowKey(event);
-    return;
-  }
+  for (var i = KEYBINDING_TYPES.length; i--;) {
+    var keyType = KEYBINDING_TYPES[i];
+    var keybinding = keybindings[keyType.type];
 
-  if (this._settings.forwardArrows.indexOf(arrow) !== -1) {
-    event.preventDefault();
-    this.moveFocusForward();
-    return;
-  }
+    if (!keybinding) {
+      return;
+    } else if (keybinding.constructor !== Array) {
+      keybinding = [keybinding]
+    }
 
-  if (this._settings.backArrows.indexOf(arrow) !== -1) {
-    event.preventDefault();
-    this.moveFocusBack();
-    return;
+    for (var j = keybinding.length; j--;) {
+      var key = keybinding[j]
+      if (keyCode === key.keyCode) {
+        var modifier = key.modifier;
```

Comment:
This suggests that the keybinding object is `{ keycode: x, modifer: y }`. I think it would be more flexible to allow arbitrary event properties. That way, for example, you could bind to an event that has both shift and ctrl down.

Also, some people might want to use the newer standard `key` prop instead of `keyCode`.

So instead of checking these properties by name, why loop through the binding-option's properties and verify that they are all present in the event? So if the binding option is `{ keyCode: 64 }` just check `event.keyCode === 64`; but if the binding option is `{ key: 'Space', ctrlKey: true, shiftKey: true }`, then check `event.key === 'Space && event.ctrlKey === true && event.shiftKey === true`.

Does that make sense?